ports/atmel-samd/common-hal/busio/UART.c

```diff
@@ -193,21 +206,32 @@ void common_hal_busio_uart_construct(busio_uart_obj_t *self,
     // Set pads computed for this SERCOM.
     // TXPO:
     // 0x0: TX pad 0; no RTS/CTS
-    // 0x1: TX pad 2; no RTS/CTS
-    // 0x2: TX pad 0; RTS: pad 2, CTS: pad 3 (not used by us right now)
-    // So divide by 2 to map pad to value.
+    // 0x1: resevered
```

Comment:
```suggestion
    // 0x1: reserved
```

Reply:
Thanks for catching the typo. I've pushed to resolve it, plus re-reviewed and noticed that CTS and RTS pinmux assignments were flipped versus RX/TX. That's fixed too.